.github/workflows/build.yml

```diff
@@ -143,10 +127,42 @@ jobs:
             - name: Run Psalm
               run: ./vendor/bin/psalm --output-format=github
 
+    build-phar:
+        needs: tests
+        runs-on: ubuntu-latest
+        name: Build PHAR file
+        steps:
+            - uses: actions/checkout@v3
+
+            - name: Setup PHP
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: "8.3"
+                  ini-values: "phar.readonly=0,zend.exception_ignore_args=Off"
+                  coverage: none
+                  tools: box
+
+            - name: Build the PHAR
+              run: |
+                  composer config platform.php ^7.2.34 
```

Comment:
This is failing in github actions as an invalid version - I think it needs to be just `7.2.34`  as you're faking the running PHP version rather than setting a requirement?